<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuWidget</class>
 <widget class="QWidget" name="MainMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>283</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButton_newProject">
     <property name="toolTip">
      <string>New Project</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/images/document-new.svg</normaloff>:/images/document-new.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="5">
    <widget class="QListView" name="listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolButton_openProject">
     <property name="toolTip">
      <string>Open Project</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/images/document-open.svg</normaloff>:/images/document-open.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="toolButton_closeProject">
     <property name="toolTip">
      <string>Close current project</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/images/document-close.svg</normaloff>:/images/document-close.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="toolButton_configure">
     <property name="toolTip">
      <string>Configure</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/images/configure.svg</normaloff>:/images/configure.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QToolButton" name="toolButton_help">
     <property name="toolTip">
      <string>Help</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/images/help-about.svg</normaloff>:/images/help-about.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>projectNew()</signal>
  <signal>projectOpen()</signal>
  <signal>projectClose()</signal>
  <signal>configure()</signal>
  <signal>help()</signal>
  <signal>exit()</signal>
 </slots>
</ui>
